### PR TITLE
added more explanatory detail to the docker run command description. …

### PR DESCRIPTION
**Pull request by dkp**
…Added links.

### CODE REVIEW

File: tutorial.md [hunk 1]
```diff
@@ -29,7 +31,7 @@ layout: default
   {% highlight bash %}
   ./bids_example-0.0.4.img /bids_dataset /outputs participant --participant_label 01
   {% endhighlight %}
-
+  <p>To learn more, consult this <a href=https://neurohackweek.github.io/docker-for-scientists/>tutorial</a> and watch the <a href=https://www.slideshare.net/chrisfilo1/docker-for-scientists>workshop video</a>. Additional links and tips can be found here: <a href=https://sites.google.com/a/email.arizona.edu/bmw/resources/bids>BIDS Links and Tips</a>.</p>
```
Comment (chrisfilo) [inline tutorial.md]: "to learn more about Docker"

**Comment by chrisfilo** [comment]
Closes #3

**Comment by dkp** [comment]
Added link to PLOS 2017 paper at bottom of tutorial.md

**Comment by dkp** [comment]
as requested ; )

**Comment by chrisfilo** [comment]
Thanks!

**Pull request merged**